example/user_auth/README.md

```diff
@@ -7,3 +7,18 @@ dune exec example/user_auth/main.exe
 This example implements a very simple authentication system using `Basic` authentication.
 
 The middleware stores the authenticated user in the request's context, which can be retrieved in the handlers.
+
+The username and password for the authentication are `admin` and `admin`.
+The corresponding `Basic` header is `Basic YWRtaW46YWRtaW4=`.
```

Comment:
A bit of a nitpick, but the header is `Authorization`. Since we use `curl --user` now maybe this line is unnecessary?

Reply:
It is, thanks!